projects/igniteui-angular/migrations/common/UpdateChanges.ts

```diff
@@ -34,6 +34,17 @@ export class UpdateChanges {
     public get projectService(): tss.server.ProjectService {
         if (!this._projectService) {
             this._projectService = createProjectService(this.serverHost);
+            // Force Angular service to compile project on initial load w/ configure project
+            // otherwise if the first compilation occurs on an HTML file the project won't have proper refs
+            // and no actual angular metadata will be resolved for the rest of the migration
+            const mainPath = path.resolve(this._projectService.currentDirectory, 'src/app/app.component.ts');
```

Comment:
`'src/app/app.component.ts'` this path doesn't seem right to me as the file may not be present. Why don't we force the LS to resolve to `main.ts` instead? We could also get its relative location from the `angular.json` file since I don't think hard coding the whole path is a good idea.